test/tests/u-options.json

```diff
@@ -102,6 +105,25 @@
         { "type": "bidiIsolation", "value": "\u2069" }
       ]
     },
+    {
+      "src": ".local $world = {world :string u:dir=ltr u:id=foo} {{hello {$world}}}",
```

Comment:
This is intended as the transitivity text, I think? Should we have one with mulitple `.local` declarations? or `.input` followed by `.local`?

Reply:
Eh, not sure if it's necessary. I think the important part is making sure that moving the option from the placeholder to a declaration works.